Hymns/ContentView.swift

```diff
@@ -1,33 +1,30 @@
-//
-//  ContentView.swift
-//  Hymns
-//
-//  Created by Luke Lu on 3/23/20.
-//  Copyright © 2020 skywalkerdude. All rights reserved.
-//
-
 import SwiftUI
 
 struct ContentView: View {
-    var hymns = hymnTestData
-    @State private var searchText : String = ""
-
+    @State var selected = 0
 
     var body: some View {
-        NavigationView {
-            VStack {
-                SearchBar(text: $searchText)
-                List {
-                    ForEach(hymns.filter { self.searchText.isEmpty ? true : $0.contains(self.searchText)}, id: \.self) { hymn in
-                                        Text(hymn)
-                    }
-                }.navigationBarTitle(Text("Look up any hymn"))
-            }
+        TabView(selection: $selected) {
+            //systemName: basically means they are coming from SF Icons.
+            SearchView().tabItem {
+                Image(systemName: "magnifyingglass")
+            }.tag(0)
+
+            BrowseView().tabItem {
+                Image(systemName: "book")
+            }.tag(1)
+
+            FavoritesView().tabItem {
+                Image(systemName: "heart")
+            }.tag(2)
+
+            SettingsView().tabItem {
+                Image(systemName: "gear")
+            }.tag(3)
         }
     }
 }
 
-
 struct ContentView_Previews: PreviewProvider {
     static var previews: some View {
         ContentView()
```

Comment:
Could you wrap this in a Group{} so we can see more in the preview?

```
Group { 
    ContentView(selected: 0)
    ContentView(selected: 1)
    ContentView(selected: 2)
    ContentView(selected: 3)
}
```

Reply:
Nice! I didn't know about that. I addressed in commit 7a95b7b